spec/connection_spec.rb

```diff
@@ -92,4 +92,59 @@
 
   end
 
+  describe "jdbc_uri" do
+    before(:all) { @olap_connection = Mondrian::OLAP::Connection }
+
+    describe "MS SQL jTDS driver" do
+      it "should return a valid JDBC URI" do
+        @olap_connection.new(
+          driver: 'mssql',
+          host: 'example.com',
+          port: 1234,
+          instance: 'MSSQLSERVER',
+          database: 'example_db',
+          domain: 'win_domain'
+        ).jdbc_uri.should == 'jdbc:jtds:sqlserver://example.com:1234/example_db;instance=MSSQLSERVER;domain=win_domain'
+      end
+    end
+
+    describe "SQL Server" do
+      it "should return a valid JDBC URI with port" do
+        @olap_connection.new(
+          driver: 'sqlserver',
+          host: 'example.com',
+          port: 1234,
+          instance: 'MSSQLSERVER',
```

Comment:
Why `instance` is not used in the generated `jdbc_uri` in this case?
According to https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15 it seems that both instance name and a port can be used.